Lesson26/gozde/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Image Gallery</title>
+  <link rel="stylesheet" type="text/css" href="./styles.css">
+</head>
+<body>
+  <h1>Image Gallery</h1>
+  <button id="clear-screen" class="button">Clear Screen</button>
+  <button id="fetch-button" class="button">Fetch Images</button>
+  <div id="image-container"></div>
```

Comment:
It would be better user experience if the image container is placed below slider as when many images get fetched, slider is no longer visible on the page.